feat: connector multidc on alert engine connection (#969)
gravitee-io/gravitee-docs: pages/ae/am/am-installation.adoc

Let's say you have to connect to multiple Alert Engines in order to propagate the triggers across multiple clusters.
The connector lets you do this:

```yaml
alerts:
  alert-engine:
    enabled: true
    engines:
      cluster1:
        endpoints:
          - https://localhost:8072/
        security:
          username: admin
          password: adminadmin
        #ssl:
        #  keystore:
        #    type: jks # Supports jks, pem, pkcs12
        #    path: /path/to/keystore.jks
        #    password: changeit
        #  truststore:
        #    type: jks # Supports jks, pem, pkcs12
        #    path: /path/to/truststore.jks
        #    password: changeit
      default:
        endpoints:
          - http://localhost:8073/
        security:
          username: admin
          password: adminadmin
        #ssl:
        #  keystore:
        #    type: pem # Supports jks, pem, pkcs12
        #    certs:
        #      - /path/to/cert.pem
        #      - /path/to/cert2.pem
        #    keys:
        #      - /path/to/key.pem
        #      - /path/to/key2.pem
        #  truststore:
        #    type: pem # Supports jks, pem, pkcs12
        #    path: /path/to/truststore.pem

> ```

Reviewer: Maybe we can mention the fact that we don't need redundancy configuration on `ws` section.
Except to overwrite the `sendEventsOnHttp` attribute ?

The connector lets you do this:

```yaml
alerts:
  alert-engine:
    enabled: true
    engines:
      cluster1:
        endpoints:
          - https://localhost:8072/
        security:
          username: admin
          password: adminadmin
        #ssl:
        #  keystore:
        #    type: jks # Supports jks, pem, pkcs12
        #    path: /path/to/keystore.jks
        #    password: changeit
        #  truststore:
        #    type: jks # Supports jks, pem, pkcs12
        #    path: /path/to/truststore.jks
        #    password: changeit
      default:
        endpoints:
          - http://localhost:8073/
        security:
          username: admin
          password: adminadmin
        #ssl:
        #  keystore:
        #    type: pem # Supports jks, pem, pkcs12
        #    certs:
        #      - /path/to/cert.pem
        #      - /path/to/cert2.pem
        #    keys:
        #      - /path/to/key.pem
        #      - /path/to/key2.pem
        #  truststore:
        #    type: pem # Supports jks, pem, pkcs12
        #    path: /path/to/truststore.pem
```

You can add as many `alerts.alert-engine.engines.<cluster-name>` as possible to connect to your Alert Engines.

> NOTE: You will only be able to connect to multiple clusters to send triggers but this not the case on the gateway to send

Reviewer 2: ```suggestion
NOTE: You can send triggers to multiple clusters, but on the gateway, events are only handled by `alerts.alert-engine.engines.default`.
```

> events, the latter being handled by `alerts.alert-engine.engines.default`.

Reviewer 2: ```suggestion
```

> WARNING: `alerts.alert-engine.engines.default` is a mandatory cluster to add in both management-api and gateway configuration.

Reviewer 2: ```suggestion
WARNING: `alerts.alert-engine.engines.default` is required in both the management-api and gateway configurations.
```
